apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -279,6 +279,9 @@ suite('conversation.js >', function() {
       container.innerHTML = innerHTML;
       // This crudely emulates the CSS styles applied to the message list
       container.lastElementChild.style.paddingBottom = '16px';
+
+      // force reflow
+      container.scrollTop;
```

Comment:
I'm curious, was you able to reproduce this issue somehow? I doubt that green Treeherder is the right indicator for this issue - I've just created "fake" PR and re-run scrolling test ~60 times - and all are green - https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f86ada32e5f5bafd57448d29715ed66c2b710eae (it's in GU20 in my PR) :)